gulpfile.js

```diff
@@ -74,16 +74,15 @@ gulp.task('scripts', ['clean'], function() {
 
 gulp.task('styles', ['clean'], function() {
 
-  return gulp.src('src/common.css')
+  return gulp.src(['src/common.css'], {base: 'src'})
+    .pipe($.sourcemaps.init())
     .pipe($.header(config.banner, {
       timestamp: (new Date()).toISOString(), pkg: config.pkg
     }))
     .pipe($.concat('select.css'))
-    .pipe(gulp.dest('dist'))
```

Comment:
@wesleycho I've had to restore this line locally to get `select.css` saved to dist, are you sure it should have been removed?